modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/SqlRowHandler.java

```diff
@@ -186,15 +185,15 @@ public abstract static class RowWrapper {
 
         abstract @Nullable Object get(int field);
 
-        abstract void set(int field, @Nullable Object value);
-
         abstract BinaryTuple toBinaryTuple();
     }
 
     /**
      * Wrapper over an array of objects.
      */
     private static class ObjectsArrayRowWrapper extends RowWrapper {
+        private static final ObjectsArrayRowWrapper EMPTY_ROW = new ObjectsArrayRowWrapper(RowSchema.builder().build(), new Object[0]);
```

Comment:
I suggest moving it into RowBuilderImpl and replace `ObjectsArrayRowWrapper.EMPTY_ROW` with `EMPTY_ROW`